admin-cli/src/index.ts

```diff
@@ -121,32 +120,76 @@ namespace v3v4 {
     }
   }
 
-  // "Auto-" accept upgrade proposals.
-  export const accept = async (ledgerParams: LedgerParams, company: string) => {
+  export const upgradeAccept = async (ledgerParams: LedgerParams, company: string) => {
     const companyLedger = connect(ledgerParams, company);
-    const employees = await companyLedger.query(davl3.EmployeeRole, {});
-    for (const employeeRole of employees) {
-      const employee = employeeRole.key;
+    const employeeRoleContracts = await companyLedger.query(davl3.EmployeeRole, {});
+    // Create update agreements.
+    for (const employeeRoleContract of employeeRoleContracts) {
+      const employee = employeeRoleContract.key;
       const employeeLedger = connect(ledgerParams, employee);
-      const [upgradeProposal] =
-        await employeeLedger.query(davlUpgradev3v4.UpgradeProposal, { employee: employee });
-      await employeeLedger.exercise(
+      const [upgradeProposalContract] =
+        await employeeLedger.query(davlUpgradev3v4.UpgradeProposal, { employee });
+        await employeeLedger.exercise(
```

Comment:
```suggestion
      await employeeLedger.exercise(
```
That indentation is confusing.